features/step_definitions/i18n_steps.rb

```diff
@@ -1,3 +1,11 @@
+Given /^String "([^"]*)" corresponds to "([^"]*)"$/ do |translation, key|
+  *seq, last_key = key.split('.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.